pre_commit_all_the_way_down/python_doc.py

```diff
@@ -263,10 +274,14 @@ def print_errors(errors: List[Error]):
         msg = []
         ctx = error.context
         msg.append(f"Error in {ctx.filename}:{ctx.start}:{ctx.end}")
-        msg.append("STDOUT was:")
-        msg.append(indent(error.stdout.rstrip(), " | ", predicate=lambda _line: True))
-        msg.append("STDERR was:")
-        msg.append(indent(error.stderr.rstrip(), " | ", predicate=lambda _line: True))
+        stdout = error.stdout.rstrip()
+        if stdout.strip():
+            msg.append("STDOUT was:")
+            msg.append(indent(stdout, " | ", predicate=lambda _line: True))
+        stderr = error.stderr.rstrip()
+        if stderr.strip():
```

Comment:
why the .strip here ?

Reply:
To prevent printing an empty stderr / stdout

Reply:
oh fair enough. I think I actually read it as "split"